addons/stock/views/stock_orderpoint_views.xml

```diff
@@ -44,8 +44,9 @@
                 <field name="product_category_id" column_invisible="True"/>
                 <field name="product_tmpl_id" column_invisible="True"/>
                 <field name="unwanted_replenish" column_invisible="True"/>
+                <field name="replenish_location_ids" column_invisible="True"/>
                 <field name="product_id" readonly="product_id" force_save="1" context="{'default_detailed_type': 'product'}"/>
-                <field name="location_id" options="{'no_create': True}" groups="stock.group_stock_multi_locations"/>
+                <field name="location_id" options="{'no_create': True}" groups="stock.group_stock_multi_locations" domain="[('id', 'in', replenish_location_ids)]"/>
```

Comment:
It's absolutely not the same purpose... The replenish location was made to search automatically for missing products in those locations. It should be possible to add an exception on orderpoint. If you want to limit the location maybe we could add the internal type
@thomasdeleval

Reply:
@thomasdeleval, seeking your confirmation with regards to this in addition to a few prior points

Thanks 😇

Reply:
@rare-odoo indeed better to just have a domain on internal locations

Reply:
Thank you for the response